frontstage/views/__init__.py

```diff
@@ -10,12 +10,27 @@
 from frontstage.views.info import info_bp
 
 
+# Import endpoints and register blueprints
```

Comment:
Why aren't these imports at the top?

Reply:
:+1: originally I thought that the blueprints had to be registered before we import the endpoints so this is leftover from that